### ISSUE TITLE
[Feature Request] Radius Compensation (Hole and Perimeters)

### ISSUE THREAD

**Issue by MarioMass** [alexrj/Slic3r#2065]
I actually spent the last week to try to have a right radius in my printed models but any (any) time i have right dimensions but wrong hole radius.
My first thought was that the slicer not compute the layer with and made the circumference of the radius without offset given that many holes are smaller exactly of the layer width.
Searching on google i found that this is a popular problem of all 3d printers.
Considering that the radius error is directly proportional of the number of triangular face of inner perimeter of the hole i suggest to add ad "scale up factor" to correct this problem as an advanced option.
This can help small hole to be correctly dimensioned; an other thing is on a large and detailed hole.

I do a number of test but, actually, i can't say the right ratio; (may this can change on different printers).

In short:
A factor to cale up all hole.
